### ISSUE TITLE
Remove P2A leftovers

### ISSUE THREAD

**Issue by fmarier** [brave/brave-browser#42644]
P2A (Privacy-Preserving Ad Analytics) is now disabled via Griffin for all users, but the client code is [still present](https://github.com/search?q=repo%3Abrave%2Fbrave-core%20P2A&type=code) in brave-core. It's also in our [pin list](https://github.com/brave/brave-core/blob/78c5424b75906dfc85d2483303d2427428840757/chromium_src/net/tools/transport_security_state_generator/input_file_parsers.cc#L69-L78).

Once that's done, we can close all [open issues](https://github.com/brave/brave-browser/labels/p2a) and update the [P2A wiki page](https://github.com/brave/brave-browser/wiki/Randomized-Response-for-Private-Advertising-Analytics).

**Comment by tredondo** [comment]
Does this explain p2a-json.brave.com not being listed in [brave_network_audit_allowed_lists.h](https://github.com/brave/brave-core/blob/master/browser/net/brave_network_audit_allowed_lists.h)? See #47279.

**Comment by fmarier** [comment]
@tredondo Yes, P2A has been inactive for some time and so it's no longer necessary for it to be in that list.

**Comment by tredondo** [comment]
What does "inactive" mean? Brave still attempts to connect to [p2a-json.brave.com](https://p2a-json.brave.com).

**Comment by fmarier** [comment]
@tredondo Which operating system are you using? If Linux, what's your distribution and how did you install Brave?

**Comment by tredondo** [comment]
Ubuntu 25.04, installed from the apt source per the instructions at https://brave.com/linux/#debian-ubuntu-mint

**Comment by fmarier** [comment]
This was removed in the 1.81 branch (currently in Beta) and so you'll see it completely gone in the next major release if you're on the release channel: https://github.com/brave/brave-browser/issues/42644#issuecomment-3030199594

**Comment by fmarier** [comment]
It should have been disabled before that via Griffin / variations though. It's possible there was a glitch there, I'm not sure. Do you see the following in the last section of `brave://version`?

```
BraveP3ATypicalJSONDeprecationEnabled:Enabled
```

In any case, the code is all gone in Beta.
